canvas/gradient.go

```diff
@@ -73,6 +73,16 @@ func (g *LinearGradient) Move(pos fyne.Position) {
 	repaint(g)
 }
 
+// Resize the gradient to a new size. Causes a refresh, as cached textures become invalid on resize.
```

Comment:
Ah, sorry I have one more suggestion - the second sentence about refreshing should just be moved to an inline code comment just above the g.Refresh() call, as it's more of an internal implementation detail than an API documentation. (And in RadialGradient too)